news/20200306.misc

```diff
@@ -1 +1 @@
-Simplify HTMFileContentsParser into two functions
\ No newline at end of file
+Simplify HTMFileContentsParser into two functions
```

Comment:
🤔

Reply:
Rebase problem. Not sure how that sneaked itself in!